src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -1431,7 +1431,7 @@ Copyright (c) .NET Foundation. All rights reserved.
   <Import Project="$(MSBuildThisFileDirectory)Microsoft.NET.ObsoleteReferences.targets" />
   <Import Project="$(MSBuildThisFileDirectory)Microsoft.NET.Publish.targets" />
   <Import Project="$(MSBuildThisFileDirectory)Microsoft.NET.PackStubs.targets" Condition="('$(Language)' == 'C++' and '$(_EnablePackageReferencesInVCProjects)' != 'true')"/>
-  <Import Project="$(MSBuildThisFileDirectory)Microsoft.NET.PackTool.targets" />
+  <Import Project="$(MSBuildThisFileDirectory)Microsoft.NET.PackTool.targets" Condition="'$(PackAsTool)' == 'true'" />
```

Comment:
Adding this conditional means that we can modify things to our heart's content inside the PackTool.targets and not worry about corrupting non-Tool projects' state.